README.md

```diff
@@ -196,8 +193,8 @@ metadata:
 spec:
   description: string # optional up to 1050 characters
   service: string # name of the service to associate this SLO with, may refer (depends on implementation) to existing object Kind: Service
-  indicator: # see SLI below for details
```

Comment:
Not against the change, but it does break backward compatibility.

Would this change mean the spec moves to 2.0?